libs/barista-components/experimental/drawer-table/README.md

```diff
@@ -0,0 +1,39 @@
+# Drawer table
+
```

Comment:
@ffriedl89 should we add a experimental notice here as well?

Reply:
Thanks for spotting this - done!